.changes/v2.22.0/607-features.md

```diff
@@ -0,0 +1,7 @@
+* Added `firmware` field to `VmSpecSection` type and `BootOptions` to `Vm` type [GH-607]
```

Comment:
It should be capital "Firmware", doesn't it?

Reply:
Yes, updated.